app/views/tickets/index.html.erb

```diff
@@ -50,8 +50,8 @@
             <% end %>
 
             <td class="<%= ticket.status %>-ticket"><%= pretty_ticket_status(ticket) %></td>
-            <td style="cursor:help;" title="<%= ticket.updated_at.strftime("%b %d, %Y %I:%M %p") %>"><%= time_ago_in_words(ticket.updated_at) %> ago</td>
```

Comment:
Just use the way every other place display time, there's a helper method for that. That would be the proper way

Reply:
> I would have used `time_ago_in_words_tagged` for [this], but that would slightly change the way it works currently. Right now, the title is on the entire column.

Reply:
I can't imagine any reason why it should be the way it is currently specifically. Kira was probably just in a rush to get it all working. Sets/Takedowns don't adhere to the general style of the codebase as well. That results in these slight discrepancies with the time formatting you noticed.

Reply:
Standardization gets no complaints from me